ext/nnstreamer/android_source/gstamcsrc_looper.cc

```diff
@@ -17,48 +17,50 @@
 /**
  * @file	  gstamcsrc_looper.cc
  * @date	  19 May 2019
- * @brief	  A looper thread to perform event messages between amcsrc and media codec
+ * @brief	  A looper thread to perform event messages between amcsrc and media
+ * codec
```

Comment:
Q: Did you intend this change? (applying clang format to doxygen comment block)

Reply:
No. That's what clang-format does. We need to adjust clang-format configuration in the future: 1. 80-col limit --> 120-col limit? 2. header include ordering

Reply:
How about setting `CommentPragmas`?
https://stackoverflow.com/questions/48016641/force-clang-format-to-respect-doxygen-block-comments

Reply:
That didn't help. I gotta find other options.